Content: Unfettered campaign Act 1 (#6416)
endless-sky/endless-sky: data/hai/unfettered side missions.txt

			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
				`	(Never.)`
					decline
			`	You head to his throne room to see him, and this time the Unfettered guards have no issue with letting you pass. It is only once you enter the room that you see the void left by his absence. One of his aids comes your way, equipped with a translator. When you ask where he might be they think a bit and come up with the only logical answer, "Go look on Firelode, on top of the [biggest, most prominent] mountain. That is where he usually is at this [time, day]."`
				accept
	to fail
		has "unfettered: outcast"

mission "<uhai> 13: Better Translator"
	landing
	name "Speak Sestor"
	description "Go to <stopovers> to acquire the records of Sestor language."
	source "Firelode"
	stopover "Hai-home"
	destination "Darkcloak"
	passengers 1
	to offer
		has "<uhai> 13: Better Translator Directions: offered"
	on offer
		conversation
			`	You climb on top of the greatest mountain for hours, having been told that using a ship or any other means of transportation would make you be judged unworthy. Choot'k is indeed there, on the top, admiring the view.`
			scene "scene/chootk firelode"

> 			`	He notices your arrival and invites you to admire the view as well with a geasture of the hand. He seems glad to see you. "<first>! I often come to this place, to meditate..." he says, before turning to take a good look at you. "It's been a while, but I keep hearing rumors about you. One day you may even threaten to take my place..." You see his grip around his blade tightening. For a few seconds you can feel some tension, but then he laughs.`

Reviewer: ```suggestion
			`	He notices your arrival and invites you to admire the view as well with a gesture of the hand. He seems glad to see you. "<first>! I often come to this place, to meditate..." he says, before turning to take a good look at you. "It's been a while, but I keep hearing rumors about you. One day you may even threaten to take my place..." You see his grip around his blade tightening. For a few seconds you can feel some tension, but then he laughs.`
```

			choice
				`	"How can I learn Hai?"`
				`	"Is there some kind of translator?"`
					goto translator
			`	"Well, to be frank, we don't have the ability to teach all of its nuances to you in time. However, we believe that our brethren do have some way of doing that, given how apt they are with languages and how keen they are to meet new races."`
			label translator
			`	"We think our Hai brethren may indeed have what we need, but we need to find a way to obtain it.`
			`	"They are not ready to meet the birds yet.`
				to display
					not "event: hai-human resolution announced"
			`	"Perhaps you can go to them to get a translator, although that is not the preferred way. But maybe they are ready to meet the birds now.`
				to display
					has "event: hai-human resolution announced"

> 			`	"In the meantime, you should go and meet the aliens up North and speak with them, representing us peacefully. They may not allow you to land on most of their planets. They may have grown distrusful of newcomers - probably because of our own actions. Go and see how pathethic these birds are, and then you should go to He-Yi about that translator."`

Reviewer: ```suggestion
			`	"In the meantime, you should go and meet the aliens up North and speak with them, representing us peacefully. They may not allow you to land on most of their planets. They may have grown distrustful of newcomers - probably because of our own actions. Go and see how pathetic these birds are, and then you should go to He-Yi about that translator."`
```

				`	(Later.)`
					defer
			`	You arrive to He-Yi standing proudly in front of a statue in his war room. "I had no idea you were a disciple of Tejolis as well."`
			choice
				`	"What do you mean?"`
				`	(Let him continue.)`
			`	"I have discovered that you have somehow acquired an item I myself thought only a thing of legends. Something so [precious, rare] and yet so evasive. I want it. I am of course referring to the 'cloaking device.' I consider this item of the highest [importance, priority], and I am willing to give you <payment> for it. I will not lie to you: this item nearly [guarantees, ensures] the survival of the one who has it if he knows how to use it properly. It would also allow new strategies in our war against the bird people."`
			choice
				`	"I accept."`
					accept
				`	"I don't want to trade now, but maybe later."`
				`	"I refuse."`
					decline
			`	"Very well. I [would not, do not want to] take it from you, of course. If you change your mind, my offer still stands."`
				defer
	on accept
		payment 50000000
		outfit "Cloaking Device" -1
		log `Gave a cloaking device to He-Yi for a very big sum. It should keep him safe.`
		"unfettered rep: He-Yi" ++

> mission "<uhai> 20: Bigeminate Sigil: Boarding Equipement"

Reviewer: ```suggestion
mission "<uhai> 20: Bigeminate Sigil: Boarding Equipment"
```
(Please fix this anywhere else the mission is used)

		fine 100000000
	on decline
		"reputation: Hai (Unfettered)" <?= -1

mission "Unfettered: Si-Ra's Trust Broken"
	invisible
	landing
	to offer
		has "Unfettered: Prove Your Strength: active"
		"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 <= 12
	to fail
		"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 > 12
	on fail
		dialog "You have broken the trust of the Unfettered captain by trying to come to a duel with too many ships. He will not show up again."
		fail "Unfettered: Prove Your Strength"
	to complete
		has "Unfettered: Prove Your Strength: active"

mission "Unfettered: Attracted Attention"
	landing
	invisible
	source "Freedom"
	destination "Darkcloak"
	to offer
		has "Unfettered: Prove Your Strength 2: active"
		has "outfit (installed): Jump Drive"
	on enter "Almaaz"

> 		dialog `The Unfettered "pirate" spits a a string of old-timey nautical curses, but one word, which you can barely hear him mutter, stands out: "Cerberus." He then comes closer to address you directly, "There even more of these scurvy bilge rats than I expected. I warned ye about coming here with a jump drive!"`

Reviewer 2: ```suggestion
		dialog `The Unfettered "pirate" spits a a string of old-timey nautical curses, but one word, which you can barely hear him mutter, stands out: "Cerberus." He then comes closer to address you directly, "There be even more of these scurvy bilge rats than I expected. I warned ye about coming here with a jump drive!"`
```

		fine 100000000
	on decline
		"reputation: Hai (Unfettered)" <?= -1

mission "Unfettered: Si-Ra's Trust Broken"
	invisible
	landing
	to offer
		has "Unfettered: Prove Your Strength: active"
		"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 <= 12
	to fail
		"ships: Space Liner" * 2 + "ships: Transport" * 4 + "ships: Light Freighter" * 2 + "ships: Heavy Freighter" * 4 + "ships: Utility" * 6 + "ships: Interceptor" * 2 + "ships: Light Warship" * 4 + "ships: Medium Warship" * 6 + "ships: Heavy Warship" * 12 > 12
	on fail
		dialog "You have broken the trust of the Unfettered captain by trying to come to a duel with too many ships. He will not show up again."
		fail "Unfettered: Prove Your Strength"
	to complete
		has "Unfettered: Prove Your Strength: active"

mission "Unfettered: Attracted Attention"
	landing
	invisible
	source "Freedom"
	destination "Darkcloak"
	to offer
		has "Unfettered: Prove Your Strength 2: active"
		has "outfit (installed): Jump Drive"
	on enter "Almaaz"

> 		dialog `The Unfettered "pirate" spits a a string of old-timey nautical curses, but one word, which you can barely hear him mutter, stands out: "Cerberus." He then comes closer to address you directly, "There even more of these scurvy bilge rats than I expected. I warned ye about coming here with a jump drive!"`

Author: I think he shouldnt have perfect English but yeah up to @EjoThims as well

> 		dialog `The Unfettered "pirate" spits a a string of old-timey nautical curses, but one word, which you can barely hear him mutter, stands out: "Cerberus." He then comes closer to address you directly, "There even more of these scurvy bilge rats than I expected. I warned ye about coming here with a jump drive!"`

Reviewer 2: Whoops

> 		dialog `The Unfettered "pirate" spits a a string of old-timey nautical curses,  but one word, which you can barely hear him mutter, stands out: "Cerberus." He then comes closer to address you directly, "There even more of these scurvy bilge rats than I expected. I warned ye about coming here with a jump drive!"`

Reviewer: ```suggestion
		dialog `The Unfettered "pirate" spits a a string of old-timey nautical curses, but one word, which you can barely hear him mutter, stands out: "Cerberus." He then comes closer to address you directly, "There even more of these scurvy bilge rats than I expected. I warned ye about coming here with a jump drive!"`
```

	npc kill
		government "Alpha"
		personality entering uninterested fleeing timid
		ship "Alpha Auxiliary (Bringing Supplies)" "Settler"
		conversation "unfettered: destroyed auxiliary"

mission "Unfettered: Alpha Leaving With Supplies"
	invisible
	minor
	repeat
	landing
	source "Warfeed"
	deadline
	to offer
		not "Unfettered: Alphas: Find The Auxiliary: offered"
		not "unfettered: destroyed auxiliary"
		not "Unfettered: Alpha Bringing Supplies: active"
		not "Unfettered: Alpha Leaving With Supplies: active"
		"reputation: Hai (Unfettered)" >= 0
		or
			not "unfettered: saw auxiliary"
			random < 5
	on offer
		set "unfettered: saw auxiliary"
	on enter "Hi Yahr"
		dialog `As you launch, the local authorities hail you, "An important guest is leaving with you. They seem to have activated their weapon systems... Get out, fast. And do not interfere with that launching ship."`
	npc kill
		government "Alpha"
		personality launching uninterested fleeing timid
		ship "Alpha Auxiliary (Leaving With Supplies)" "Settler"
		conversation "unfettered: destroyed auxiliary"

> # TEMPORARILY DISABLED UNTIL HR SITUATION WITH PIRATES IS CLEARED OUT

Author: I've been waiting on Grey for this
It isnt critical for this story though

> # TEMPORARILY DISABLED UNTIL HR SITUATION WITH PIRATES IS CLEARED OUT

Author: I'lll remove it from the campaign, we'll do it in a separate PR, eventually.

# mission "Unfettered: Pirate Help"
# 	landing
# 	name `Help the pirates.`

> # 	description `Once you are strong enough, go to Hai-home and convince the governor to let you help them with their pirate problem. You will instead find a way to help the pirates.`

Author: Yeah it'll be changed with HR because they'll come and find you for it instead of just saying there may be an attack

		`"Bondir" Atomic Thruster`
		`"Bufaer" Atomic Steering`
		"Hyperdrive"
		"Quantum Keystone"

mission "Unfettered: Ionic Prize"
	landing
	source "Darkcloak"
	to offer
		has "Unfettered: Earn Your Prize: done"
		has "event: unfettered: disable-er"
	on offer
		conversation "modified shield beetle prize"
		give ship "Modified Shield Beetle (Ionic Turret)" "Basilisk"
		fail

event "unfettered: syscore's got it"

ship "Modified Shield Beetle" "Modified Shield Beetle (Large Systems Core)"
	outfits
		"Ionic Blaster Prototype" 4
		"Hai Tracker Pod" 4
		"Ionic Turret Prototype" 2
		"Chameleon Anti-Missile"
		"Boulder Reactor" 2
		"Hai Ravine Batteries"
		"Hai Diamond Regenerator"
		"Systems Core (Large)"
		"Hai Zephyr Cooling" 3
		"Hai Tracker" 224
		"Outfits Expansion"
		`"Bondir" Atomic Thruster`
		`"Bufaer" Atomic Steering`
		"Hyperdrive"
		"Quantum Keystone"

mission "Unfettered: Immortal Prize"
	landing
	source "Darkcloak"
	to offer
		has "Unfettered: Earn Your Prize: done"
		has "event: unfettered: syscore's got it"
	on offer
		conversation "modified shield beetle prize"
		give ship "Modified Shield Beetle (Large Systems Core)" "Basilisk"
		fail

conversation "more staves"

> 	`As you wonder in the spaceport, you see He-Yi's herald, or at least he has the same mask. He seems interested to know your progress in acquiring weapons.`

Reviewer: ```suggestion
	`As you wander in the spaceport, you see He-Yi's herald, or at least someone with the same mask. He seems interested to know your progress in acquiring weapons.`
```

	choice
		`	"Could you look at what I have here to tell me if it's what you're looking for?"`
		`	"It's going fine, thank you for asking."`
			goto fine

> 	`	You bring him to your ship and he sees a weapon that appears to be interesting. "This... <staff> staff seems to be adequate for my master. We would only require more of these."`

Reviewer: What's `<staff>`?

Author: Pug staves or coalition ones

	label fine
	`	He wishes you good luck and hopes you can find the staves. He reminds you a large amount of them is requested - twenty, to be precise.`

mission "Bring More Enforcer Staves"
	minor # so both staff conversations don't pop up at the same time
	invisible
	source "Warfeed"
	substitutions
		<staff> "Enforcer"
	to offer
		has "Unfettered: Close Quarter: offered"
		not "Bring Coalition Weapons: offered"
		not "Bring Pug Staves: offered"
	on offer
		require "Enforcer Riot Staff"
		conversation "more staves"

mission "Bring More Pug Staves"
	minor
	invisible
	source "Warfeed"
	substitutions
		<staff> "Pug Peacekeeping"
	to offer
		has "Unfettered: Close Quarter: offered"
		not "Bring Coalition Weapons: offered"
		not "Bring Pug Staves: offered"
	on offer
		require "Pug Peacekeeping Staff"
		conversation "more staves"

mission "Bring Pug Staves"
	landing
	invisible
	source "Warfeed"
	to offer
		has "Unfettered: Close Quarter: offered"
	on offer

> 		dialog `Do you want to sell 20 Pug Peacekeeping Staves to the Unfettered? He Yi says they can give you 2500000 for it.`

Reviewer: ```suggestion
		dialog `Do you want to sell 20 Pug Peacekeeping Staffs to the Unfettered? He-Yi says they can give you 2500000 for it.`
```

	on accept
		payment 2500000
		outfit "Pug Peacekeeping Staff" -20

> 		dialog `The Unfettered thank you for bringing these 20 staves to them. They give you <payment>.`

Reviewer: ```suggestion
		dialog `The Unfettered thank you for bringing these 20 staffs to them. They give you <payment>.`
```

		fail

mission "Bring Coalition Weapons"
	landing
	invisible
	source "Warfeed"
	to offer
		has "Unfettered: Close Quarter: offered"
	on offer

> 		dialog `Do you want to sell 20 Enforcer Riot Staves to the Unfettered? He Yi says they can give you 4000000 for it.`

Reviewer: ```suggestion
		dialog `Do you want to sell 20 Enforcer Riot Staffs to the Unfettered? He-Yi says they can give you 4000000 for it.`
```
What's with the staves?

	on accept
		payment 4000000
		outfit "Enforcer Riot Staff" -20

> 		dialog `The Unfettered thank you for bringing these 20 staves to them. They give you <payment>.`

Reviewer: ```suggestion
		dialog `The Unfettered thank you for bringing these 20 staffs to them. They give you <payment>.`
```

				goto stories
			label lucky
			action
				outfit "Dual Pulse Pistols"
				set "unfettered: won the duel"
			`	You actually manage to reach cover, thanks to him stumbling a bit on the approach. He was probably anxious to beat you... Now that you are behind cover, you have a strategic advantage, and he misses his shots. It seems that the pistol quickly runs out of power, and using this opportunity you shoot back at him. Most of your shots miss, but because he cannot shoot back and you are well protected anyway, you can take the time to aim precisely. You manage to shoot his arm and he drops his pistol. He does not shout at all when getting shot, simply sits on the ground on that position, showing he surrenders. You get out of your cover and grab the other pistol. "You can keep them; you've earned it. This was a fair fight, Captain! I'd like to try again some time, and maybe get my pistols back..." He then goes away and sits on the side, probably thinking about what mistakes he made.`
				goto heyi
			label ready

> 			`	You try to shoot him, but he's too fast and you too inexperienced. You have made yourself an easy target for him by standing still; you fall as you receive a shot to your shoulder. It hurts, but nothing seems to be broken, and soon after you cannot feel anything in your arm at all. You opponent comes to you and helps you to get up. "Pain is a good teacher sometimes," he says.`

Reviewer: ```suggestion
			`	You try to shoot him, but he's too fast and you are too inexperienced. You have made yourself an easy target for him by standing still; you fall as you receive a shot to your shoulder. It hurts, but nothing seems to be broken, and soon after you cannot feel anything in your arm at all. You opponent comes to you and helps you to get up. "Pain is a good teacher sometimes," he says.`
```

			label stories

> 			`	 As agreed, you speak to him of many stories, including how you happened to find Hai space, and how you joined the Unfettered. He seems captivated by all of it, and so are his friends. Then he goes back to fight with his friends, seemingly in teams.`

Reviewer: ```suggestion
			`	As agreed, you speak to him of many stories, including how you happened to find Hai space, and how you joined the Unfettered. He seems captivated by all of it, and so are his friends. Then he goes back to fight with his friends, seemingly in teams.`
```

event "unfettered: better sestor translator"

# no blocked dialog because 1 bunk, and it's a side mission anyway
mission "Unfettered: Speak Sestor Notice"
	minor
	name "Find Choot'k"
	description "Go to <destination> to find Choot'k. He should be on top of the biggest mountain."
	source "Darkcloak"
	destination "Firelode"
	to offer
		has "salary: Unfettered"
		has "Hai Reveal [C01] Cleanup: done"
		or
			has "Unfettered: Solifuge 8: Useful Korath Scientists: offered"
			random < 15
	on offer
		conversation
			`Do you want to go see Choot'k, to talk with him of the past and maybe see if you could do something for him?`
				to display
					not "Unfettered: Solifuge 8: Useful Korath Scientists: offered"
			`Do you want to go see if Choot'k can do something to improve the translator, as Mera suggested?`
				to display
					has "Unfettered: Solifuge 8: Useful Korath Scientists: offered"
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
				`	(Never.)`
					decline

> 			`	You head to his throne room to see him, and this time the Unfettered gards have no issue with letting you pass. It is only once you enter the room you learn he is missing from one of his aid. They tell you to go look on Firelode, on top of the biggest mountain, as that's probably where he would be.`

Reviewer: ```suggestion
			`	You head to his throne room to see him, and this time the Unfettered guards have no issue with letting you pass. It is only once you enter the room you learn he is missing from one of his aid. They tell you to go look on Firelode, on top of the biggest mountain, as that's probably where he would be.`
```

				accept

mission "Unfettered: Speak Sestor"
	landing
	name "Speak Sestor"
	description "Go to <stopovers> to acquire the records of Sestor language."
	source "Firelode"
	stopover "Hai-home"
	destination "Darkcloak"
	passengers 1
	to offer
		has "Unfettered: Speak Sestor Notice: offered"
	on offer
		conversation
			`	You climb on top of the greatest mountain when you find Choot'k is indeed there, admiring the view.`
			scene "scene/chootk firelode"

> 			`	He notices your arrival and invites you to admire the view as well. He seems happy to see you. "<first>! It's been a while, but I keep hearing rumors about you. One day you may even threaten to take my place..." You see his grip around his blade tightening. For a few seconds you can feel some tension, but then he laughes.`

Reviewer: ```suggestion
			`	He notices your arrival and invites you to admire the view as well. He seems happy to see you. "<first>! It's been a while, but I keep hearing rumors about you. One day you may even threaten to take my place..." You see his grip around his blade tightening. For a few seconds you can feel some tension, but then he laughs.`
```

			`	"No, not yet, at least. Now, I would duel you if I had the time, but I have duties to attend to, which you may help me with, actually. I want to go to <stopovers> to get the records of Korath interactions before the raids. Mera has informed me that the translator - whilst better than nothing - is not sufficient for all the tasks we have in mind."`

> 			`	"I am sure you have already noticed that communication with the Koraths can be... difficult, since you met them."`

Reviewer: ```suggestion
			`	"I am sure you have already noticed that communication with the Korath can be... difficult, since you met them."`
```

# Copyright (c) 2022 by Hurleveur
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

mission "Unfettered: Neutral With The Hai"
	name `Negotiate with the Hai`

> 	description `The Unfettered have asked you to go negotiate with the Hai to stop your conflict with them. Go to <destination>.`

Author: Idk

				goto lostbars
			label wait
			`You patiently wait for the intruder to arrive and hit them in the back with your pistol. Given it is a member of your crew that betrayed you, you decide not to involve anyone else, as they could be complicit. You first remove the weapon they are carrying. After doing a thorough search, you notice a distinct marking on the crew member's leg, which is only visible by a special ultraviolet light. This seems bad, very bad. Then you pause for a second to wonder if you really want to know about this.`
			choice
				`(Ignorance is bliss.)`
				`(I prefer not to stay in the dark. Wake them.)`
					goto light
			`You throw a bucket of water on the crew member, waking them up.`
			label bliss
			`You leave the traitor here on this world, and they seem thankful but deeply troubled by what happened. You wonder what could have attracted such attention to you. Whatever it was it should probably be safely stashed away, or gotten rid off.`
				accept
			label light

> 			`You throw a bucket of water on the crew member. They then wake up, and at first refuses to tell you anything, but once you reveal you're aware of their marking, they agrees to reveal something to you. They seem to recognize the hopelessness of their situation, and maybe willing to bargain. "This sign means that I holds my life to someone, and have to do one task, any task, to repay them. There is something here, on this ship, of the highest importance to some... I may not, and will not say anything else, no matter what you try to make me do it. I am dead already, and they own me."`

Author: Concede to reveal smth to you then?

			`You nod, and they then proceed, "I will tell you the truth, but then you will let me go and we will never see each other again. But let me be clear, I will not speak, under any circumstances other than those."`
			choice
				`(This is it. I want to know all.)`
				`(Ignorance is bliss.)`
					goto bliss

> 			`They takes a deep breath. "An important organization wants these gold bars you have on board. They're not just normal gold. If you were to sell them to me, I could give you a very generous price of 1000000 credits per bar, and you wouldn't be bothered by this again. 1 million per bar. You see, something is within these bars, something of the highest importance, and they want all of them. They will not stop until they have every single one. I may have told you too much for them to allow me to live already."`

Author: Like you waiting hours for the crew member to reveal answers? Not sure what can happen honnestly
There was a narrative but I had to remove it (old friend soldier that'd help you out with this)

> 			`They takes a deep breath. "An important organization wants these gold bars you have on board. They're not just normal gold. If you were to sell them to me, I could give you a very generous price of 1000000 credits per bar, and you wouldn't be bothered by this again. 1 million per bar. You see, something is within these bars, something of the highest importance, and they want all of them. They will not stop until they have every single one. I may have told you too much for them to allow me to live already."`

Reviewer: I mean, it is weird that they tell you stuff, and without any narrated change of tone or something they just suddenly start regretting it and become overly cautious.

> 			`They takes a deep breath. "An important organization wants these gold bars you have on board. They're not just normal gold. If you were to sell them to me, I could give you a very generous price of 1000000 credits per bar, and you wouldn't be bothered by this again. 1 million per bar. You see, something is within these bars, something of the highest importance, and they want all of them. They will not stop until they have every single one. I may have told you too much for them to allow me to live already."`

Author: The idea is that they see their old plan failed and try to save their life.
Failure is death, and so is revealing too much info to the enemy but if you manage to bring back something... Then you can live

				decline

mission "Unfettered: Pirate Help"
	landing
	name `Help the pirates.`
	description `Once you are strong enough, go to Hai-home and convince the governor to let you help them with their pirate problem. You will instead find a way to help the pirates.`
	source "Warfeed"
	stopover "Allhome"
	to offer
		has "Unfettered: Escort Hai Fleet: done"
		not "Pirate Troubles [0]: offered"
		has "reputation: Hai" >= 0
	on offer
		conversation
			`Entering the spaceport, you quickly find Nimo. He looks relieved you are here and that you have arrived safe and well with the convoy. It seems like he speaks to you with more respect than before. "Finally, you have arrived. When I heard about the unfettered human attack, I was worried sick."`
			choice
				`	"It takes more than a few pirates to scare me off."`
					goto strong
				`	"I almost did not make it back alive. That was a tough battle."`
				`	"We had to run for our lives."`
					goto run
			`	"Close call, I see. Maybe the Hai should assign a slightly larger fleet next time... Or maybe you yourself need more ships."`
				goto attacks
			label run

> 			`	"Oh, I may have overestimated your strength then. You should wait a while before doing what I am about to ask, because it will require some strength."`

Author: Prob could change the first one by 'abilities'?

> 			`	"Oh, I may have overestimated your strength then. You should wait a while before doing what I am about to ask, because it will require some strength."`

Reviewer: Works for me.

			choice
				`	"How will I convince the Hai I'm there to help them?"`
			`	"I hear those unfettered humans attack are very frequent these days... And that planet is on the border of Hai space. It is a very likely target for any pirates. I also have information indicating an attack will come, soon. It is easy to arrange this. Help them repel it and they will most probably trust you with the job. But this is not an urgent mission at all, <first>. You may return to your previous occupations if you do not feel strong enough yet."`
			choice
				`	"I will go to Allhome and help them repel this attack then, once I am strong enough to repel a pirate fleet."`
					accept
				`	"I'm not interested in doing this."`
					decline
	to fail
		has "defeated scar's legion"
	on fail

> 		dialog `You have decided to side with the Hai on this. This was not really an official mission so the Unfettered will not be mad about it. Still, do not expect any payment from them.`

Author: Sure, but just to be clear you told the UHai you would pay the tribute and then do the opposite
This is just meant to be a "does not break the campaign" kind of fail

> 		dialog `You have decided to side with the Hai on this. This was not really an official mission so the Unfettered will not be mad about it. Still, do not expect any payment from them.`

Reviewer: I guess I wasn't paying enough attention then.

					accept
				`	"Sorry, but I have other things to do."`
			branch "too late"
				"unfettered: days" - "unfettered: startdays south" > 150 + random
			`	They talk for a bit, and after a while still decide to come with you, for better or worse.`
				decline
			label "too late"
			action
				"unfettered: missing southern hai" ++
			`	They talk for a bit, and after a while decide not to come with you, given the time they had to wait for you to come here and pick them up.`

ship "Corvette" "Corvette (Harpy)"
	outfits
		"Hai Tracker Pod" 4
		"Hai Tracker" 112
		"Heavy Anti-Missile Turret" 2
		"Fission Reactor"
		"LP036a Battery Pack"
		"D94-YV Shield Generator"
		"Cooling Ducts"
		"Bunk Room"
		"Pulse Rifle" 3
		"Brig" 2
		`"Biroo" Atomic Thruster`
		`"Bondir" Atomic Steering`
		"Hyperdrive"
	cargo
		outfits
			"Gold Bars" 15

mission "Unfettered: Family Hunt From Newhome"
	name "Family Hunt"
	description "A Hai family has been on a pirate ship labeled as a Merchant, thinking it was their due transport. Find it and board it to save them."
	landing
	source "Newhome"
	destination "Firelode"
	to offer
		has "Unfettered: The Word Has Spread: active"
	on offer
		conversation

> 			`You search all over the spaceport but cannot find any trace of the Hai you are looking for. Looking at the news you learn the Hai family you were sent to recover have been missing for a while. Apparently they have been taken by a pirate who pretended to be a merchant, ready to bring them home. The name of the ship is 'The Harpy', and it probably still is in Hai space. You launch as soon as possible to find it, as rumors say that ship is as fast as light itself.`

Author: I didnt know what else to compare it with
Maybe give the name of a specie on the hai world as an expression?

> 			`You search all over the spaceport but cannot find any trace of the Hai you are looking for. Looking at the news you learn the Hai family you were sent to recover have been missing for a while. Apparently they have been taken by a pirate who pretended to be a merchant, ready to bring them home. The name of the ship is 'The Harpy', and it probably still is in Hai space. You launch as soon as possible to find it, as rumors say that ship is as fast as light itself.`

Reviewer: Yes, that would work better.

				launch
	npc board
		government "Bounty (Disguised)"
		personality coward uninterested
		system
			distance 1 2
		ship "Corvette (Harpy)" "The Harpy"
		conversation
			action
				payment 250000

> 			`You manage to find the Hai family, thankfully no harm has come to them. They do not seem to be in shock but still insist to give you all that they have, <payment>, to thank you. They say that after this event, they're even more looking forward to meeting their cousins, and learning more about them (you assume they refer to the Unfettered.)`

Author: Yeah I could remove it but idk if it'll be clear enough
Mb change it to their brethren

> 			`You manage to find the Hai family, thankfully no harm has come to them. They do not seem to be in shock but still insist to give you all that they have, <payment>, to thank you. They say that after this event, they're even more looking forward to meeting their cousins, and learning more about them (you assume they refer to the Unfettered.)`

Reviewer: idk, maybe `cousins up north` would be enough

			`	The mother adds, "Some Hai ships tried to help us, or at least that is what I assumed when feeling the small shocks to the ship, but they were the exception. We will probably be safer in unfettered space, where they don't let pirates roam freely, whether they are disguised as merchants or not."`
				decline
	on fail
		conversation
			`The Unfettered will not be happy about the loss of these young Hai, as they care a lot for their reputation on the Hai side. It will probably be deduced from your payment.`
			action
				"unfettered: missing southern hai" += 2

mission "Unfettered: The Word Has Spread South: Greenbloom"
	invisible
	landing
	source "Greenbloom"
	destination "Firelode"
	to offer
		has "Unfettered: The Word Has Spread: active"
	on offer
		conversation

> 			`These 17 Hai calmly enter your ship. Most of them do not look like warriors. You see one of them drop a drawing without noticing.`

Author: Because the bunks already got booked by starting the mission

> 			`These 17 Hai calmly enter your ship. Most of them do not look like warriors. You see one of them drop a drawing without noticing.`

Reviewer: Oh ok then.

ship "Marauder Leviathan" "Leviathan (Cube)"
	outfits
		"A370 Atomic Thruster"
		"A525 Atomic Steering"
		"Anti-Missile Turret"
		"D94-YV Shield Generator"
		"Fusion Reactor"
		Hyperdrive
		"LP144a Battery Pack"
		"Laser Rifle" 69
		"Security Station" 5
		"Small Bunk Room"
		"Liquid Helium Cooler"
		"Luxury Accommodations"
		"Proton Turret" 3
		Torpedo 102
		"Torpedo Launcher" 4
		"Torpedo Storage Rack" 6
	cargo
		outfits
			"Gold Bars" 15
			"Mystery Cube"

mission "Unfettered: Special Cargo"
	name `Mystery item`
	description `Go to <waypoints> and find a way to acquire a "unique" item from the prominent warlord there. His ship is the Degenerate. Then bring the item to He-Yi on <planet>.`
	to offer
		has "Unfettered: Solifuge 9: Shield Beetles Mass Production: done"
		has "reputation: Hai" >= 0
	source "Darkcloak"
	waypoint "Alcyone"
	destination "Warfeed"
	on offer
		conversation
			`You are starting to get quite used to Unfettered spaceports: the dilapidation of different non-essential structures is beginning to feel normal. You feel like a change of scenery, so you wander to a side of the spaceport you have not explored before, but it turns out not to be very different.`

> 			`	Your thoughts are interrupted by a local Unfettered wearing a strange mask of some kind of bird, who seems to have a very important message to give you given how fast he is speaking. "He-Yi has asked me to deliver you a message! It has come to his attention that an item of great importance to us is currently in the possession of an unfettered human warlord." You guess that he means a pirate leader. "The warlord can be found in <waypoints>. We want you to go and retrieve that item right now - 'by any means,' he says."`

Author: Idk prob an error

	on complete
		require "Mystery Cube"

mission "Unfettered: Special Cargo 1B"
	invisible
	landing
	source "Stormhold"
	destination "Warfeed"
	to offer
		has "Unfettered: Special Cargo: active"
		has "reputation: Hai" >= 0
	on offer
		require "Mystery Cube" 0
		conversation
			`You land on the planet next to another empty spot, hoping the Degenerate will land there. He does not seem to want to take any chances, and lands near a few of his own warships. As you depart the ship, your hairs stand up on the back of your neck. You try to get close to the Degenerate without drawing too much attention to yourself, and eventually arrive.`
			choice
				`	(Try to steal the item.)`
					goto steal
				`	(Try to bargain.)`
					goto bargain
				`	(Try to take an isolated pirate captive and ask for the item in exchange.)`
			`	In front of the ship there's a group of pirates, and you see one of them separate from the others to go within the ship. Judging from his appearance and the way others where addressing him, you assume him to be the leader of that small band, at least. You sneak past the group in front and follow him.`
			branch luckyish
				random < 20

> 			`	You get close up behind him and are about to pull your gun... when he suddenly turns back and throws the hardest haymaker you've ever experienced! You drop like a sack to the floor and never wake up again. That pirate probably has had some experience with backstabs before.`

Reviewer: ```suggestion
			`	You get close up behind him and are about to pull your gun... when he suddenly turns back and throws the hardest haymaker you've ever experienced! You drop like a sack to the floor and never wake up again. That pirate probably has had some experience with backstabbings before.`
```

				die
			label luckyish

> 			`	You get close up behind him and are about to pull your gun... when he draws a sudden breath and begins to turn around! Fortunately, you're quicker, and you have your firearm pointed before he can do anything. Obviously experienced with hostage situations and prioritizing his own safety, he doesn't try to call for help. You present yourself to the group outside, with the gun still pointed at your hostage. The pirates look at each other and begin laughing before you die in a barrage of laserfire from the group, annihilating you and your hostage. It seems that they couldn't care less about their companions' lives.`

Reviewer: ```suggestion
			`	You get close up behind him and are about to pull your gun... when he draws a sudden breath and begins to turn around! Fortunately, you're quicker, and you have your firearm pointed at him before he can do anything. Obviously experienced with hostage situations and prioritizing his own safety, he doesn't try to call for help. You present yourself to the group outside, with the gun still pointed at your hostage. The pirates look at each other and begin laughing before you die in a barrage of laserfire from the group, annihilating you and your hostage. It seems that they couldn't care less about their companions' lives.`
```

				die
			label steal
			`	You manage to enter the ship discreetly. Some people will always notice, but you try to pose as a spaceport worker there to assist with the repairs, using equipment you find lying around. You should be able to go pretty much anywhere on the ship. You don't have any idea which part of the ship is actually damaged though, so your cover could easily be blown. There is no going back now, except maybe if you choose to provoke the warlord into a fight to get the item.`
			choice
				`	(Search the lower decks.)`
				`	(Search close to the weapons stations.)`
					goto weapons
				`	(Search the bridge.)`
					goto bridge
				`	(Look near the engines.)`
					goto engine
				`	(Check among the cargo.)`
					goto cargo
				`	(Search the upper decks.)`

> 				`	(Leave the ship before you get caught, and provoke the warlord.)`

Reviewer: ```suggestion
				`	(Leave the ship before you get caught and provoke the warlord.)`
```

					goto challenge
			`	You notice a lot of security stations around, they obviously care a lot about their warlord's security.`
				goto caught
			label weapons

> 			`	This part of the ship is mostly full of torpedoes - apparently the warlord loves them. But there are not many people here, since this part of the ship seems to have suffered no damage. The torpedoes will probably be restocked soon, so you leave before someone comes in and notices you.`

Reviewer: 'this part of the ship' is repeated too often.

				goto found
			label bridge
			`	This bridge is very luxurious, you see that this warlord likes comfort, for herself at least. But you cannot find anything that looks unique at all.`
				goto caught
			label cargo

> 			`	There are a lot of exotic things here, most of which are probably not legal at all. You better choose what to take with you, and fast! You also notice the presence of gold bars. There's no way you could carry all of them back with you discreetly, but you can drop them off the ship and then go grab them once you are out, hoping nobody will notice. Looking through a porthole you don't see anyone around at the moment. Conveniently, there is even some sand on the ground there, so the bars should sink in, partially hidden under it.`

Reviewer: ```suggestion
			`	There are a lot of exotic things here, most of which are probably not legal at all. You better choose what to take with you, and fast! You also notice the presence of gold bars. There's no way you could carry all of them back with you discreetly, but you can drop them off the ship and then go grab them once you are out, hoping nobody will notice. Looking through a porthole you don't see anyone around at the moment. Conveniently, there is even some sand on the ground there, so the bars should sink in, and be harder to notice.`
```

			branch knowsbars
				has "Unfettered: Betrayal For The Bars: declined"

> 			`	It may not be what you're looking for, but these bars are probably worth a small fortune, and you reckon you could get away with a few tons of them. They have special markings on them and are probably worth more than the value of the gold alone! You can always take out the pirate lord's ship afterward and claim them as your own, as they are more heavy and take more space than any of the other items which could interest the warlord.`

Reviewer: Just casually walk away with a couple tons of gold. This feels like minecraft :)

Author: I think I make the player throw them out the window and get them later

Reviewer: It's mostly fine, just weird

Author: I specified its using an antigrav unit that you do this, and the help of a crew member

				goto fake
			label knowsbars
			`	You recognize the same bars that earned you betrayal from a member of your crew. Something about them must make them special and hugely valuable to those who know of it.`
				goto fake
			label engine
			`	These are some fine last-generation atomic engines, a true beauty in your eyes. But admiring the engines doesn't help you find the item at all. Why would there be anything near the engines?`
				goto caught
			label fake
			choice
				`	(Take a heavily-modified alien looking pistol.)`
				`	(Take a strong personal shield.)`
				`	(Take what looks like a bomb.)`
				`	(Drop some bars on the ground to take them later.)`
					goto takebars
			label wrongitem

> 			`	You take the item, hoping it is the right one, and swiftly leave the ship. If only you had been given a more precise description, you could know for sure. Some pirates seem to notice you, and maybe they suspect something or recognize this item, but nobody tries to stop you. As you reach your ship, you notice that the Degenerate just launched, right before you. They must have noticed the item was missing.`

Reviewer: ```suggestion
			`	You take the item, hoping it is the right one, and swiftly leave the ship. If only you had been given a more precise description, you could know for sure. Some pirates seem to notice you, and maybe they suspect something or recognize this item, but nobody tries to stop you. As you reach your ship, you notice that the Degenerate just launched, right before you. They must have noticed the item was missing.`
```
I thought it has. landed already?

Reviewer: Oh I somehow managed to misread that word like 5 times, my bad

				flee
			label takebars
			`	How many stacks of bars will you take? The more, the bigger the risk and reward...`
			choice
				`	(One will do.)`
					goto one
				`	(Three should be enough.)`
					goto three
				`	(Five.)`
					goto five
				`	(Eight could be reasonable.)`
				`	(Take as much as I can: fifteen!.)`
				`	(Never mind, this isn't a good idea.)`
					goto fake
			`	You got too greedy. As you're dropping the bars, you feel something cold hit your head. You never wake up again.`
				die
			label one
			action
				outfit "Gold Bars"

> 			``

Reviewer: Why are the quotes there?

Author: Its needed to have a goto

			``
				goto dropbars
			label five
			action
				outfit "Gold Bars" 5
			label dropbars
			`	You drop the gold and then leave the ship. Afterwards, you go around the ship to the location where they are partially buried in the sand, and take them discreetly with some members of your crew. You have to silence one or two curious pirates, but that's about it, and you can now head back to your ship. As expected, they have noticed the theft. you see them launching before you can do so yourself. This will be a bumpy ride.`
				flee
			label caught
			`	You do not manage to find the item here. However, you've attracted way too much suspicion by looking and opening crates everywhere you could. One local worker asks you something about mechanics. Being no engineer yourself, you cannot make any sense of what he says, even less formulate a correct response, and he calls a guard over you. You are captured and sold as a slave, condemned to a few years of hard work before a sad death.`
				die
			label found
			choice
				`	(Take an alien-looking modified torpedo warhead.)`
					goto stupid
				`	(Take an alien skull.)`
					goto wrongitem
				`	(Take a weird-looking small cube.)`

> 			`	Surprisingly, you appear to have found what you were looking for: a small cube, completely regular and seemingly alien in design. Maybe the warlord was not aware of its value, considering she placed it here, without much protection or even trying to hide it. This has to be the item that He-Yi wants, so you take it with you. Its small size makes it easy to conceal as you make your way off the ship and back to your own. This probably will not go unnoticed for long however, so you launch as soon as you can. The original pirate fleet is already heading your way as you depart the atmosphere.`

Reviewer: ```suggestion
			`	Surprisingly, you appear to have found what you were looking for: a small cube, completely regular and seemingly alien in design. Maybe the warlord was not aware of its value, considering she placed it here, without much protection or even trying to hide it. This has to be the item that He-Yi wants, so you take it with you. Its small size makes it easy to conceal as you make your way off the ship and back to your own. This probably will not go unnoticed for long however, so you launch as soon as you can. The original pirate fleet is already heading your way as you are leaving the atmosphere.`
```

			choice
				`	"That low? Well, here you go mate."`
				`	"I don't have that kind of money. Maybe we could negotiate?"`
					goto negotiate
				`	"There's no way I'm paying so much for an item I haven't even seen! I challenge your warlord to a fight."`
					goto challenge
			`	He seems disappointed, but he shows you to the item, a little cube, and you bring it back to your ship without any trouble. He also signals the local warships belonging to the warlord to let you leave without trouble.`
			action
				payment -10000000

> 			``

Reviewer: ?

Author: Whats the issue? The `` is needed for the goto and he's disapointed caus he realizes he could have sold it much more

Reviewer: oh I didn't realise that an empty quote could be necessary

				goto leaveinpeace
			label negotiate
			`	"I can cut it down by... 2 million. Last chance, boy. 8 million credits, what do ya say?"`
			choice
				`	"Deal!"`

> 				`	""There's no way I'm paying so much for an item I haven't even seen! I challenge your warlord to a fight.""`

Reviewer: ```suggestion
				`	"There's no way I'm paying so much for an item I haven't even seen! I challenge your warlord to a fight."`
```

				set "unfettered: brokered deal"
				"reputation: Mhairi's Legion" >?= 1
			``
				accept
			label challenge
			`	You launch your fleet and send a challenge to the warlord, putting her authority and competence in question. She cannot resist the urge to come and kill you, you only hope you can beat her and collect the item from her ship.`
				flee
	on accept
		outfit "Mystery Cube"

mission "Unfettered: Special Cargo 2"
	landing
	name `Mystery Gathering`
	description `Collect a tribute of food from <stopovers>, and place the mystery cube without anyone noticing.`
	source "Warfeed"
	stopover "Mirrorlake"
	cargo "food (tribute)" 93
	blocked `You need <tons> of cargo space for the next Unfettered mission.`
	to offer
		has "reputation: Hai" >= 0
		has "Unfettered: Special Cargo: done"
	on offer
		conversation
			branch bought
				has "unfettered: brokered deal"
			branch stole
				has "Unfettered: Special Cargo 1B: done"
			action
				log `Recovered a mystery cube from a pirate ship in the heat of battle.`
			``
				goto afterlog
			label stole
			action
				log `Managed to steal a very precious mystery cube from a pirate ship.`
			``
				goto afterlog
			label bought
			action

> 				log `Bought a mystery cube off pirates for the Unfettered. Price tag was large.`

Reviewer: ```suggestion
				log `Bought a mystery cube off pirates for the Unfettered. The price tag was large.`
```

			label afterlog

> 			`You find He-Yi in his war room, which is full of strange objects - only half of which appear to be weapons. You notice some keystone carvings, but also a statue, probably of some Hai goddess. He seems to have multiple missives, serving as his eyes and ears everywhere. You bring him the cube. "Very well done! But this is not all. You will have a big [reward, payment] once the mission is completed."`

Reviewer: > He seems to have multiple missives, serving as his eyes and ears everywhere.

What do you mean?

Author: Well the warlord has apprentices/acolytes that tell him all thats going on

Reviewer: https://www.oxfordlearnersdictionaries.com/definition/english/missive?q=missive

Author: Right, I'll change it to disciples

			`	You thank him for telling you this story and then think to yourself. This means there is a good chance the Pug have the non-lethal weapons you are looking for. Now you just have to find the Pug, but they'll probably not give you these weapons willingly...`

mission "Unfettered: Wanderer Homeworld"
	invisible
	source "Firelode"
	destination "Vara K'chrai"
	to offer
		has "Unfettered: Solifuge 9: Shield Beetles Mass Production: done"
	to fail
		has "Unfettered: Speak Hai: failed"
	waypoint "Ka'ch'chrai"
	on waypoint

> 		dialog `Judging by the population compared to the other worlds, this appears to be the capital of these aliens. You should land to introduce yourself as an diplomat between them and the Hai.`

Reviewer: ```suggestion
		dialog `Judging by the population compared to the other worlds, this appears to be the capital of these aliens. You should land to introduce yourself as a diplomat between them and the Hai.`
```

mission "Unfettered: Speak Hai"
	name `Meet the Wanderers and acquire a translator`
	description `You need to be able to act as a diplomat between the Hai and the aliens up north. First go and meet those aliens. Then go to <stopovers> to see if He-Yi will provide a translator for you.`
	source "Firelode"
	stopover "Warfeed"
	to offer
		has "Unfettered: Solifuge 9: Shield Beetles Mass Production: done"
		not "language: Wanderer"
	on offer
		conversation

> 			`Moving through the spaceport, you notice some buildings are not as dilapidated as before. You wonder if the Unfettered are the ones who invested into repairs, or if the new Hai arrivals brought this about. Whatever the case is, this does not look like a place of conflict anymore. It looks like a place of hope.`

Reviewer: You really love `dilapidated`, don't you? It might still be better to change it to something else in some of the descriptions.

Author: Its not me its a reviewer
Yeah idk I dont know synonyms or anything

Author: I used synonyms from online

> 			`	An Unfettered herald - recognizable with his peculiar bird mask - approaches and notices you looking around. "Incredible, is it not? To see hope on the face of buildings and on the face of people. I am here to say that it's time you learn how to communicate with us - with the True language of the Hai. Not only with us. You also need to be able to make demands to the birds, and there is only one appropriate language for it. We have learned that they too speak Hai, even if they disgrace our language by doing so."`

Reviewer: ```suggestion
			`	An Unfettered herald - recognizable with his peculiar bird mask - approaches and notices you looking around. "Incredible, is it not? To see hope on the face of buildings and on the face of people. I am here to say that it's time you learn how to communicate with us - with the True language of the Hai. You also need to be able to make demands to the birds, and there is only one appropriate language for it. We have learned that they too speak Hai, even if they disgrace our language by doing so."`
```

			choice
				`	"How can I learn Hai?"`
				`	"Is there some kind of translator?"`
					goto translator

> 			`	"Well to be frank, we don't have to ability to teach all of its nuances to you in time. However, we believe that our brethren do have some way of doing that, given how apt they are with languages and how keen they are to meet new races."`

Reviewer: ```suggestion
			`	"Well, to be frank, we don't have the ability to teach all of its nuances to you in time. However, we believe that our brethren do have some way of doing that, given how apt they are with languages and how keen they are to meet new races."`
```

			action
				"reputation: Wanderer" = 0
			label neutral
			choice
				`	"I guess I'll try to land on their planet."`
					accept
				`	"I am not interested in being able to communicate with them."`
			`	"Well if you want to work with us, you'll have to set that preference aside. And don't forget: the translator is for communicating with us as well. Some among the most conservative of us despise speaking in your language. This will be a token of good faith on your part."`
			choice
				`	"I guess I'll try to land on their planet."`
					accept
	on visit
		dialog "You have not yet met the Wanderers and acquired a translator from <stopovers> to communicate with them. "
	to complete
		has "language: Wanderer"
	on complete

> 		log `The Unfettered asked me to be able to speak like a True Hai, because that is the language in which I should make demands to the Unfettered later.`

Reviewer: To the Unfettered?

Author: To the wanderers right
Also to the uhai but not that much

		dialog `You land and find the herald you spoke to earlier. "We now know we can speak to you in the True language of the Hai, finally. And you can speak to us in Hai, through the translator. Most of us will be willing to continue communicating in your language, but expect to use the translator to speak and understand our Chiefs. They'll probably act like it's nothing, but trust me, they care."`

mission "Unfettered: Bring Detainer"
	landing
	name `Recover a detainer`

> 	description `He-Yi has asked you to earn a translation device by bringing back a powerful alien weapon, said to be able to neutralize the engines of its target. You should probably look around Kor Sestor space.`

Reviewer: Kor Sestor is not mentioned anywhere.

Author: Right, space east or smth would be better

				`	"Sure, I will go and bring it back for you."`
					accept
				`	"I doubt I have the strength to face the Korath machines."`
			`	"If you cannot acquire the weapon, then you do not deserve this device. There should be other ways to get it than brute force, however. You could loot a ship while its allies are distracted. Use whatever tools you have at your disposal. What do you say?"`
			choice
				`	"Sure, I will go and bring it back for you."`
					accept
				`	"This is way too risky. I'm out."`
			action
				fail "Unfettered: Know Your Enemy"
	to fail
		has "language: Wanderer"
	on visit
		dialog `You do not yet have the weapon He-Yi asked for. Investigate the battles of the Korath automata and you may see it in action.`
	on complete
		outfit "Korath Detainer" -1
		set "language: Wanderer"
		dialog `He-Yi is impressed that you managed to bring this detainer to him, and happily exchanges it for a translator. He says you should still go to his herald to bring him news of this, and the scans for a reward.`

mission "Unfettered: Found Detainer"
	boarding
	invisible
	to offer
		has "Unfettered: Bring Detainer: active"
	source
		outfits "Korath Detainer"
	on offer
		conversation

> 			`"The 'Detainer' on this ship matches He-Yi's description perfectly! Quickly, grab it and get out of here."`

Reviewer: ```suggestion
			`The 'Detainer' on this ship matches He-Yi's description perfectly! Quickly, grab it and get out of here.`
```

	destination "Warfeed"
	to offer
		has "language: Wanderer"
		has "Unfettered: Solifuge 9: Shield Beetles Mass Production: done"
		not "Unfettered: Speak Hai: active"
	on offer
		log "Minor People" "Mysterious Unfettered Herald" "A herald of He-Yi that does not want to reveal his true identity. He seems to have a lot of culture, and stays calm in most situations."
		conversation
			`A herald comes to your ship as you exit it. He is still wearing that same mask, although you feel like the color has slightly changed, "Hello again, <last>, my master has yet another task for you before the war starts."`
			choice
				`	"I am listening."`
					goto task
				`	"First I have a question, why have you not introduced yourself yet?"`
				`	"Later."`
					defer
			`	"When I wear that mask, I may not reveal my identity to anyone, that is my burden as a herald of He-Yi. I do not wear this mask all the time, of course, only when I am on mission for him, or in his war room."`
			label task
			`	"We want you to survey all systems belonging to our enemies, to assess if they have some backup fleets or potential allies nearby."`
			branch neutral
				has "reputation: Wanderer" >= 0
			choice

> 				`	"But I am at war with them It would be at a risk to me."`

Reviewer: ```suggestion
				`	"But I am at war with them. It would be at a risk for me."`
```

> 			`	He seems a bit angry, "Again? In the last mission we had arranged for them to not shoot at you, what did you do?? Well there is nothing we can do for you this time, I just hope you have some mean of hiding, or a very agile ship. Theirs are not that impressive but they do have long range missiles, meaning you should probably install some Chameleon defenses on your ship."`

Reviewer: ```suggestion
			`	He seems a bit angry, "Again? In the last mission we had arranged for them to not shoot at you, what did you do?? Well there is nothing we can do for you this time, I just hope you have some means of hiding, or a very agile ship. Theirs are not that impressive but they do have long range missiles, meaning you should probably install some Chameleon defenses on your ship."`
```

			label neutral
			choice
				`	"I'm in."`
					accept
				`	"Why can't you do it yourselves?"`
			`	"He-Yi is currently busy, too busy to go do a survey in his personal ship, and he will not let anyone else fly it, including me. That means we have no way to do the scouting ourselves without risking lives."`
			choice
				`	"Got it, you need me. I'll do it then."`
					accept
				`	"Well you'll have to wait for him to find the time, I'm not interested in taking this kind of risk."`
					decline
	waypoint "Chy'chra"
	waypoint "Pik'ro'iyak"
	waypoint "Ik'kara'ka"
	waypoint "Chirr'ay'akai"
	waypoint "Sich'ka'ara"
	waypoint "Kiro'ku"
	waypoint "Ka'pru"
	waypoint "Ap'arak"
	waypoint "Ka'ch'chrai"
	waypoint "Ek'kek'ru"
	waypoint "Si'yak'ku"
	waypoint "Iyech'yek"
	waypoint "Kiru'kichi"
	waypoint "Es'sprak'ai"
	waypoint "Pug Iyik"
	on waypoint
		dialog `You have scouted all of the Wanderer's systems. It's time to come back to <destination> for your reward.`
	on complete
		payment 1000000
		conversation

> 			`The herald is there when you land, apparently waiting for you. "My master could not come to receive the scans himself, but be sure I shall give them to him. Please, accept these <payment> credits in his name. I think you deserve a small explanation as to why these aliens are hostile to us. As you probably already know, we are planning to invade them shortly. But to be fair, we have already started doing a few skirmishes with them, notably for testing their strength, but mostly to get food from them. They have huge ships transporting a thousand tons of cargo"`

Reviewer: ```suggestion
			`The herald is there when you land, apparently waiting for you. "My master could not come to receive the scans himself, but be sure I shall give them to him. Please, accept these <payment> credits in his name. I think you deserve a small explanation as to why these aliens are hostile to us. As you probably already know, we are planning to invade them shortly. But, to be fair, we have already started doing a few skirmishes with them, notably for testing their strength, but mostly to get food from them. They have huge ships transporting a thousand tons of cargo"`
```

			branch "know about the Pug"
				has "main plot completed"
			choice
				`	"Speak to him about the weird aliens."`
					goto warn
				`	"Do not mention that to him."`
					decline
			label warn
			action
				log `I warned the Unfettered about the weird aliens north, they did not seem to care.`
			`	You explain to the herald you have seen some weird aliens north, that could be allied to the Wanderers. Looking at the brief scans you took of them he seems to discard that warning as if it was nothing, "I do not know how those aliens have obtained jump drives, they most have stolen them somehow. Once we have taken Wanderer space they will prove to be a very handy supply of new jump drives..."`
				decline
			label "know about the Pug"
			choice
				`	"Speak to him about the Pug."`
					goto warn
				`	"Do not mention that to him."`
					decline

> 			`	You explain to the herald that you have seen weird aliens north, the Pug, which have had dealing with humanity before. He does not trust you when you speak of technology capable of breaking down hyperlanes, "there is no way an alien race with such primitive ships would have that kind of technology. They must have simply abused your primitive sensors. They must have stolen the jump drives, and I only see that as an opportunity for later." You try to convince him he should not take them lightly but he does not want to listen. "We have already done too much to prepare for war to turn back at the first sign of potential trouble."`

Reviewer: ```suggestion
			`	You explain to the herald that you have seen weird aliens north, the Pug, which have had dealing with humanity before. He does not trust you when you speak of technology capable of breaking down hyperlanes, "there is no way an alien race with such primitive ships would have that kind of technology. They must have simply abused your primitive sensors. They must have stolen the jump drives, and I only see that as an opportunity for later." You try to convince him he should not take them lightly, but he does not want to listen. "We have already done too much to prepare for war to turn back at the first sign of potential trouble."`
```

			action
				log `I warned the Unfettered about the Pug near the Wanderer, they did not seem to care.`

mission "Unfettered: Cloaking Interest"
	invisible
	minor
	source "Warfeed"
	to offer
		has "Unfettered: Special Cargo 2: done"
		random < 20
	on offer
		require "Cloaking Device"
		conversation
			`A herald tells you He-Yi wants to see you again in his war room for a very generous proposition. He also mentions it's not urgent, so you can come back later if you're busy.`
			choice
				`	(Comply with the request.)`
				`	(Later.)`
					defer
			`	You arrive to He-Yi standing proudly in front of a statue in his war room. "I had no idea you were a disciple of Tejolis as well."`
			choice
				`	"What do you mean?"`
				`	(Let him continue speaking.)`

> 			`	"I have discovered that you have somehow acquired an item I myself thought only a thing of legends. Something so [precious, rare] and yet so evasive. I want it. I am of course referring to the 'cloaking device.' I consider this item of the highest [importance, priority], and I am willing to give you 50 million for it. I will not lie to you: this item nearly [guarantees, ensures] the survival of the one who has it if he knows how to use it properly. It would also allow new strategies in our war against the bird people."`

Reviewer: ```suggestion
			`	"I have discovered that you have somehow acquired an item I myself thought only a thing of legends. Something so [precious, rare] and yet so evasive. I want it. I am of course referring to the 'cloaking device.' I consider this item of the highest [importance, priority], and I am willing to give you 50 million credits for it. I will not lie to you: this item nearly [guarantees, ensures] the survival of the one who has it if he knows how to use it properly. It would also allow new strategies in our war against the bird people."`
```

			choice
				`	"I accept."`
					accept
				`	"I don't want to trade now, but maybe later."`
				`	"I refuse."`
					decline
			`	"Very well. I [would not, do not want to] take it from you, of course. If you change your mind, my offer still stands."`
				defer
	on accept
		payment 50000000
		outfit "Cloaking Device" -1
		log `I gave my cloaking device to He-Yi for a very big sum. It should keep him safe.`

conversation "more staves"
	`As you wonder in the spaceport, you see He-Yi's herald, or at least he has the same mask. He seems interested to know your progress in acquiring weapons.`
	choice
		`	"Could you look at what I have here to tell me if it's what you're looking for?"`
		`	"It's going fine, thank you for asking."`
			goto fine

> 	`	You bring him to your ship and he sees a weapon that appears of interest. "This... <staff> staff seems to be adequate for what my master wants. We would only require more of these."`

Reviewer: ```suggestion
	`	You bring him to your ship and he sees a weapon that appears to be interesting. "This... <staff> staff seems to be adequate for my master. We would only require more of these."`
```

mission "Bring More Enforcer Staves"
	minor # so both staff conversations don't pop up at the same time
	invisible
	source "Warfeed"
	substitutions
		<staff> "Enforcer"
	to offer
		has "Unfettered: Close Quarter: offered"
		not "Bring Coalition Weapons: offered"
		not "Bring Pug Staves: offered"
	on offer
		require "Enforcer Riot Staff"
		conversation "more staves"

mission "Bring More Pug Staves"
	invisible
	source "Warfeed"
	substitutions
		<staff> "Pug Peacekeeping"
	to offer
		has "Unfettered: Close Quarter: offered"
		not "Bring Coalition Weapons: offered"
		not "Bring Pug Staves: offered"
	on offer
		require "Pug Peacekeeping Staff"
		conversation "more staves"

mission "Bring Void Rifle"
	invisible
	source "Warfeed"
	to offer
		has "Unfettered: Close Quarter: offered"
	on offer
		require "Void Rifle" 1
		conversation
			`He-Yi seems very, very interested in this weapon. "Where did you find it? Can you bring us more? We would gladly make you rich if you do. This is the perfect weapon for training."`
			choice
				`	"Sorry, this is one of a kind."`
			`	"I can cancel the exchange then, if it is precious to your eyes."`
			choice

> 				`	"10000000 is much more precious, thank you."`

Reviewer: ```suggestion
				`	"10 million credits is much more precious, thank you."`
```

mission "<uhai> 5 Annex: Carrier Analysis"
	minor
	invisible
	landing
	source "Firelode"
	to offer
		has "<uhai> 5: Auxiliary Scans: done"
		has "ship model: Carrier"
		random < 30
		not "unfettered: outcast"
	on offer
		conversation
			`As you finish the landing procedure you receive a warning from one of your first mates: a Lightning Bug is headed straight towards his Carrier, at full speed!`
				to display
					not "flagship model: Carrier"

> 			`As you finish the landing procedure you a warning siren triggers on the bridge. One of your officers warns you that a Lightning Bug is headed straight towards you, at full speed!`

Reviewer: ```suggestion
			`As you finish the landing procedure a warning siren triggers on the bridge. One of your officers warns you that a Lightning Bug is headed straight towards you, at full speed!`
```

				to display
					has "flagship model: Carrier"
			`	Fortunately it stops as soon as it enters scanning range, and begins a full sweep of your ship, before hailing you. You immediately recognize Mera. "Hey there, <first>! I was warned about a Carrier arriving," she looks to her side, probably at one member of her crew as she continues, "because someone thought it was the Giftbringer..." before looking back to you, with a smile, "Anyway, what a pleasant surprise to know it was you. I hope you dont mind me taking some scans, you'll be compensated for it naturally. In fact, I'll allow you to come with me to the lab and choose any item you desire!"`
			choice
				`	"That seems like a good deal, I'm in!"`
					goto deal

> 				`	"You can do better than scan that ship, in fact I'm willing to sell it to you."`

Reviewer: ```suggestion
				`	"Would you like to buy it instead?"`
```
Idk. Something like this could be better. ~~Also, it avoids another 'in fact'.~~

					goto sell
				`	"My privacy is very important to me. Stop that scan at once."`
			`	She seems really surprised, but gives a sign to her officers to stop the scanning. "In that case we will erase the information collected. Truly I had no idea you would see it this way. Still, this is such a shame..."`
				decline
			label deal
			action
				set "unfettered: carrier scans"

> 			`	The scan is finished in a record speed, and Mera invites you on her ship to get to the lab. When you arrive on the bridge she is alone. "I like to be able to speak freely, you see," she says as you look at the empty bridge. "You never know who is listening," she says as she sets a course. "I have something to show you. But first, tell me, how did you acquire this Carrier? We Unfettered like to share stories, sometimes around campfires..." she says before clapping her hands. Suddenly the atmosphere of the room changes, and you can see the outside of the ship as if it was no longer there.`

Reviewer: ```suggestion
			`	The scan is finished in a record speed, and Mera invites you on her ship to get to the lab. When you arrive on the bridge she is alone. "I like to be able to speak freely, you see," she says as you look at the empty bridge. "You never know who is listening," she says as she sets a course. "I have something to show you. But first, tell me, how did you acquire this Carrier? We Unfettered like to share stories, sometimes around campfires..." she says before clapping her hands. Suddenly the atmosphere of the room changes, and you can see outside of the ship as if it was no longer there.`
```

			choice
				`	"There is not much to say, I simply bought it."`
					goto bought

> 				`	"I acquired it in a grand battle for freedom!"`

Reviewer: Is this locked behind FW?

					goto freedom
				`	"How did you do that?"`
					goto explain
				`	"I prefer not to talk about it."`
			`	"Suit yourself."`
				goto testingsite
			label bought

> 			`	"I see you are very humble, captain."`

Reviewer: ```suggestion
			`	"I see you are very humble, Captain."`
```
~~idk when it's capital and when it isn't, but I think this is the correct one here?~~

				goto testingsite
			label freedom
			`	"So you are already Unfettered in your heart!" She sighs. "I hope it means you understand that we do not all desire battle, but we do see it as a necessity."`
				goto testingsite
			label explain
			`	"This?" She says, looking around. "It's just a few projectors, in truth. This was not their primary intended use, however," she says with a chuckle.`
			label testingsite

> 			`	You quickly arrive at a collection of craters, which remind you of the moon. She stops there and takes a deep breath.`

Reviewer: Do we know if FL has ever been to the moon? Also, in this context, it should probably be capitalized, or better, `Earth's moon`. (Though most moons have craters, so idk.)

			`	"I have a new gadget I wanted to show you. If you would like to do the honors, it has never been tested before!"`
			choice
				`	(Dewit!)`
					goto dewit
				`	"Are you sure it is safe?"`
			`	"I am pretty confident it will not blow us up, if that is what you are wondering!"`
			label dewit

> 			`	You press the button and the huge turret on top of the ship starts to fire in a thunder of fire and rocks exploding, the power of the shots so big that the ship's thrusters are having a very hard time maintaining the ship in position. You can hear Mera laughing. "It works! It works!" The weapon appears to be pulse based, but the projectiles are much, much bigger, and more unstable as well, it seems.`

Reviewer: ```suggestion
			`	You press the button and the huge turret mounted on the top of the ship starts to fire in a thunder of fire and rocks exploding, the power of the shots so big that the ship's thrusters are having a difficult time maintaining the ship in position. You can hear Mera laughing. "It works! It works!" The weapon appears to be pulse based, but the projectiles are much, much bigger, and more unstable as well, it seems.`
```
> starts to fire in a thunder of fire and rocks exploding

This could use some more work

Maybe something like this:

`begins firing in a stream of bright pulses, exploding the rocks beneath in a hunder of fire`

			`	"I know I said it would not blow up, but..." she starts before being interrupted by a big explosion as the ship enters the bottom of a crater. "Well, this is why the tests are done here, after all." She seems a bit demotivated by this apparent failure. "This ship was simply not able to handle the sheer amount of heat produced! I should probably work more on stabilizing it, but that would cost energy..." she continues rambling, probably having forgotten your presence.`
			choice
				`	"Was that turret my reward?"`
				`	"Can we get back to the lab?"`
					goto lab
				`	"I've seen enough for today, get me back to my ship."`
					decline
			`	"No, of course not."`
			label lab
			action
				"unfettered rep: Mera" ++
			scene "outfit/repeater turret"
			scene "outfit/repeater"

> 			`	She sets the next destination, staying quiet. You arrive at the lab in little time, and she shows you your reward. "These are very powerful weapons, however your ship must be able to handle a lot of heat to use them. They are called 'Thermal Repeaters'. I have a lot of cleanup to do after what happened, so I'll simply ask you to choose between 2 turrets, or 5 guns."`

Reviewer: ```suggestion
			`	She sets the next destination, staying quiet. You arrive at the lab in little time, and she shows you your reward. "These are very powerful weapons, however your ship must be able to handle a lot of heat to use them. They are called 'Thermal Repeaters'. I have a lot of cleanup to do after what happened, so I'll simply ask you to choose between two of these turrets, or five guns."`
```

			choice
				`	"The turrets."`
				`	"The guns."`
					goto guns
			action
				outfit "Thermal Repeater Turret" 2
			`	"I knew you would say that. I'll pass the word to my engineers. Have fun with them."`
				goto wornout
			label guns
			action
				outfit "Thermal Repeater" 5
			label wornout
			`	"Really? I wasn't expecting you to choose the guns. But I guess they do have more total firepower, even if they lack the tracking."`
			`	She seems quite worn out by what happened today, and lets you find your way back to your ship.`
				decline

			label sell

> 			`	For a few seconds she just lays there, stunned, before answering, "Are.. You serious? I mean, of course you are, let me just think about this with my staff, I'll get back to you." She cuts the transmission for a few minutes, before reestablishing it with a formal offer, "I'll give you 10 millions and a few extras from our cache of Korath technology, including dreaded Piercer Missiles!`

Reviewer: ```suggestion
			`	For a few seconds she just lays there, stunned, before answering, "Are.. You serious? I mean, of course you are, let me just think about this with my staff, I'll get back to you." She cuts the transmission for a few minutes, before reestablishing it with a formal offer, "I'll give you 10 million credits and a few extras from our cache of Korath technology, including dreaded Piercer Missiles!`
```

			`	You will naturally be able to buy back the outfits that are currently equipped on your ship from the outfitter - only the hull is of interest to me."`
			`	A member of her staff murmurs something to her, and she realizes she forgot something, "I am so silly! You only have this one ship, so you would be unable to leave the planet if I take it. I'll give you an old Aphid we have lying around, that should allow you to get to Darkcloak safely, to buy a proper ship!"`
				to display
					"total ships" == 1
			choice
				`	"Deal!"`
				`	"Come to think of it, I'm not ready to separate myself from that ship yet, a scan is all I can give you."`
					goto deal
			action
				take ship "Carrier"
				set "unfettered: carrier"
				"unfettered rep: Mera" ++
				payment 10000000
				outfit "Thermal Repeater Rifle" 25
				outfit "Piercer Missile Launcher" 2
				outfit "Piercer Missile" 126
				outfit "Piercer Missile Rack" 4
				outfit "Plasma Core"
			branch "flag only"
				"total ships" <= 1

> 			`	They take the ship away whilst giving you the agreed payment and outfits, and invite you to head to the outfitter to get outfits back if you so desire.`

Reviewer: ```suggestion
			`	They take the ship away whilst giving you the agreed payment and outfits, and invite you to head to the outfitter to get your outfits back if you so desire.`
```

				`	"I acquired it in a grand battle for freedom!"`
					to display
						has "Defend Sabik: done"
					goto freedom
				`	"How did you do that?"`
					goto explain
				`	"I prefer not to talk about it."`
			`	"Suit yourself."`
				goto testingsite
			label bought
			`	"I see you are very humble, Captain."`
				goto testingsite
			label freedom
			`	"So you are already Unfettered in your heart!" She sighs. "I hope it means you understand that we do not all desire battle, but we do see it as a necessity."`
				goto testingsite
			label explain
			`	"This?" She says, looking around. "It's just a few projectors, in truth. This was not their primary intended use, however," she says with a chuckle.`
			label testingsite
			`	You quickly arrive at a collection of craters, which remind you of the Earth's moon you could see on magazines as a child. She stops there and takes a deep breath.`
			`	"I have a new gadget I wanted to show you. If you would like to do the honors, it has never been tested before!"`
			choice
				`	(Dewit!)`
					goto dewit
				`	"Are you sure it is safe?"`
			`	"I am pretty confident it will not blow us up, if that is what you are wondering!"`
			label dewit

> 			`	You press the button and the huge turret mounted on the top of the ship begins firing bright pulses, exploding the rocks beneath in a hunder of fire, the power of the shots so big that the ship's thrusters are having a difficult time maintaining the ship in position. You can hear Mera laughing. "It works! It works!" The weapon appears to be pulse based, but the projectiles are much, much bigger, and more unstable as well, it seems.`

Reviewer: ```suggestion
			`	You press the button and the huge turret mounted on the top of the ship begins firing bright pulses, exploding the rocks beneath in a thunder of fire, the power of the shots so big that the ship's thrusters are having a difficult time maintaining the ship in position. You can hear Mera laughing. "It works! It works!" The weapon appears to be pulse based, but the projectiles are much, much bigger, and more unstable as well, it seems.`
```

	on offer
		set "unfettered: saw auxiliary"
	on enter
		dialog

> 			`As you launch, the local authorities hail you, "An important guest is arriving, you should not be here when they do. Get out, fast.`

Reviewer: ```suggestion
			`As you launch, the local authorities hail you, "An important guest is arriving, you should not be here when they do. Get out, fast."`
```

mission "<uhai> 18: Hai Language"
	minor
	name `Meet and understand the Wanderers`

> 	description `You need to be able to act as a diplomat between the Hai and the aliens up north. First go and meet those aliens. Then go to <stopovers> to see if He-Yi will provide a translator for you, or get one trough other means.`

Reviewer 2: ```suggestion
	description `You need to be able to act as a diplomat between the Hai and the aliens up north. First go and meet those aliens. Then go to <stopovers> to see if He-Yi will provide a translator for you, or get one through other means.`
```

					goto translator
			`	"Well, to be frank, we don't have the ability to teach all of its nuances to you in time. However, we believe that our brethren do have some way of doing that, given how apt they are with languages and how keen they are to meet new races."`
			label translator
			`	"We think our Hai brethren may indeed have what we need, but we need to find a way to obtain it.`
			`	"They are not ready to meet the birds yet.`
				to display
					not "event: hai-human resolution announced"

> 			`	"Perhaps you can go to them to get a translator, although that is not the prefered way. But maybe they are ready to meet the birds now.`

Reviewer 2: ```suggestion
			`	"Perhaps you can go to them to get a translator, although that is not the preferred way. But maybe they are ready to meet the birds now.`
```

				to display
					not "unfettered: warned"
					not "unfettered: preparing for the arfectas"
			`	"Did you not say these Pugs, as you call them, are masters of deception? We already know that they have somehow tricked the Wanderers into blindly worshiping them. That ship could have tricked your sensors. Nevertheless you deserve a reward for all of this intel, here are <payment>."`
				to display
					has "unfettered: warned"

> 			`	Since he does not seem to believe you you show him the designs of the Arfecta you captured. You can see a mix of surprise and fear in his eyes as he takes a step back. "How did you capture such a ship? It seems that you will prove an even better ally than we had anticipated... That ship would be fitting of a warlord. You surely deserve this extra reward! Here, take these <payment>, you deserve it. We have indeed heard of these Pugs, but never did we imagine they would be able to display such strength... It is good to have an ally like you."`

Reviewer: ```suggestion
			`	Since he does not seem to believe you, you show him the designs of the Arfecta you captured. You can see a mix of surprise and fear in his eyes as he takes a step back. "How did you capture such a ship? It seems that you will prove an even better ally than we had anticipated... That ship would be fitting of a warlord. You surely deserve this extra reward! Here, take these <payment>, you deserve it. We have indeed heard of these Pugs, but never did we imagine they would be able to display such strength... It is good to have an ally like you."`
```
